aten/src/ATen/native/RNN.cpp

```diff
@@ -1817,56 +1817,37 @@ static auto registry =
     torch::RegisterOperators()
         .op("aten::quantized_lstm.input(Tensor input, Tensor[] hx, __torch__.torch.classes.rnn.CellParamsBase[] params, bool has_biases, int num_layers, float dropout, bool train, bool bidirectional, bool batch_first, *, ScalarType? dtype=None, bool use_dynamic=False) -> (Tensor, Tensor, Tensor)",
             torch::RegisterOperators::options()
-                .kernel<decltype(quantized_lstm_input), quantized_lstm_input>(
-                    DispatchKey::CPUTensorId))
+                .kernel(DispatchKey::CPUTensorId, TORCH_FN(quantized_lstm_input)))
```

Comment:
You had to add a *lot* of code to handle this case, when you could have just ported.

Reply:
For this specific case, I had to add one function with <10 lines. There's other call sites too and to support all of them, I had to add a bit more code, that's true. But porting all of them would be larger and I want to keep that a separate effort from this.